### PR TITLE
Fix for identical source/resource value for process operation on system log

### LINKED ISSUE #692: Source and Resource is same for all process based logs
**General Information**

- Environment : GKE, Minikube, k3s and VM
- Kernel version 5.13.0-40-generic

**To Reproduce**
1. Run Kubearmor in Systemd mode/k8s mode
2. Launch karmor in another tab and listen to all logs (karmor log --logFilter all)
3. Observe logs for all **Operation: Process**. 

**Expected behavior**

For process operation, the source and resource needs to be same.

**Screenshots**

The source and resource for all process operations are same/identical.

### PR DESCRIPTION
**Pull request by seswarrajan**
Signed-off-by: Eswar Rajan Subramanian <eswar@accuknox.com>

### CODE REVIEW

**Review by nam-jaehyun — APPROVED** [review]
LGTM

File: KubeArmor/monitor/processTree.go [hunk 1]
```diff
@@ -173,6 +173,23 @@ func (mon *SystemMonitor) GetCommand(containerID string, hostPid uint32) string
...
+}
+
+// GetCommandWithArgs Function
+func (mon *SystemMonitor) GetCommandWithArgs(containerID string, hostPid uint32) string {
+	ActiveHostPidMap := *(mon.ActiveHostPidMap)
+	ActivePidMapLock := *(mon.ActivePidMapLock)
+
+	ActivePidMapLock.Lock()
+	defer ActivePidMapLock.Unlock()
+
```
Comment (nyrahul) [inline KubeArmor/monitor/processTree.go:192]: I didn't understand this change!! What does this achieve?
New function GetCommandWithArgs() is introduced but not used!

**Review by nyrahul — COMMENTED** [review]
Please check my comment inline.

File: KubeArmor/monitor/processTree.go (hunk 1 above)
Comment (seswarrajan) [reply KubeArmor/monitor/processTree.go:192]: Yes @nyrahul 
From the existing function GetCommand(), we removed adding arguments to source.
If anyone requires to use args to source, then this function is introduced (GetCommandWithArgs).

It is currently unused. It is just kept for future use if needed.

Comment (nyrahul) [reply KubeArmor/monitor/processTree.go:192]: I see. Thanks

**Review by nyrahul — APPROVED** [review]
LGTM

**Pull request merged**